unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneBoundariesController/Tests/SceneBoundariesCheckerTests_DebugMode.cs

```diff
@@ -79,21 +80,29 @@ public IEnumerator ResetMaterialCorrectlyWhenInvalidEntitiesAreRemoved()
         public IEnumerator GLTFShapeIsInvalidatedWhenStartingOutOfBoundsDebugMode() { yield return SBC_Asserts.GLTFShapeIsInvalidatedWhenStartingOutOfBounds(scene); }
 
         [UnityTest]
+        [Explicit]
+        [Category("Explicit")]
         public IEnumerator NFTShapeIsInvalidatedWhenStartingOutOfBoundsDebugMode() { yield return SBC_Asserts.NFTShapeIsInvalidatedWhenStartingOutOfBounds(scene); }
 
         [UnityTest]
+        [Explicit]
+        [Category("Explicit")]
```

Comment:
This test shouldn't be explicit, as it shouldn't use the OpenSea endpoint (it only tests parametrizable shapes).